src/components/Dropdown/components/SingleValue/SingleValue.jsx

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from "react";
 import { components } from "react-select";
-import "./SingleValue.scss";
+import "./singleValue.scss";
```

Comment:
Why I don't see the renaming of file to lower case?

Reply:
@MosheZemah don't know